lang/compiler_test.go

```diff
@@ -144,6 +145,10 @@ func TestFluxCompiler(t *testing.T) {
 				t.Errorf(`unexpected value for now, want "%v", got "%v"`, tc.now, astProg.Now)
 			}
 
+			if p, ok := program.(lang.DependenciesAwareProgram); ok {
```

Comment:
we should be able to change this to executetest.NewTestExecuteDependencies() as well as below